

This patch removes a harmless duplicate assignment from the IPv6 code.

Please apply.



---

 25-akpm/security/selinux/hooks.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN security/selinux/hooks.c~selinux-remove-duplicate-assignment security/selinux/hooks.c
--- 25/security/selinux/hooks.c~selinux-remove-duplicate-assignment	2004-04-06 20:50:08.653926176 -0700
+++ 25-akpm/security/selinux/hooks.c	2004-04-06 20:50:08.659925264 -0700
@@ -2711,7 +2711,7 @@ out:
 static int selinux_parse_skb_ipv6(struct sk_buff *skb, struct avc_audit_data *ad)
 {
 	u8 nexthdr;
-	int ret, offset = skb->nh.raw - skb->data;
+	int ret, offset;
 	struct ipv6hdr ipv6h;
 
 	offset = skb->nh.raw - skb->data;

_
